src/site/shared/containers/ContextMenu/index.tsx

```diff
@@ -118,33 +140,30 @@ export const ContextMenu = ({info, paste}: Props) => {
         dispatch(CloseContextMenu());
     }
 
+
     const menu = useRef<HTMLDivElement>(null);
-    let pos = input?.getMousePos();
 
-    /* Relocate context menu to opposite side of cursor if it were to go off-screen */
-    if (isOpen) {
-        if (!menu.current)
-            throw new Error("ContextMenu failed: menu.current is null");
+    // Adjusts position of menu to keep it on screen
+    let pos = V(posX, posY);
```

Comment:
Call this like `menuPos` or something, also it can be `const` since the variable itself isn't changing